build.gradle

```diff
@@ -858,6 +858,7 @@ project(':core') {
     implementation project(':group-coordinator')
     implementation project(':metadata')
     implementation project(':storage:api')
+    implementation project(':tools:tools-api')
```

Comment:
Gradle can't distinguish the subproject name (https://github.com/gradle/gradle/issues/847), so I add the prefix `tools` to avoid conflict to `storage:api`

Reply:
Why does core depend on tools-api?